### PR TITLE
DICT-446 leeuwenhoek other_clinical_attribute updates

### PR DESCRIPTION
**Pull request by bullenc**
https://gdc-ctds.atlassian.net/browse/DICT-446

### CODE REVIEW

File: src/gdcdictionary/schemas/other_clinical_attribute.yaml [hunk 1]
```diff
@@ -394,7 +405,6 @@ properties:
     $ref:
       - "_terms.yaml#/dlco_ref_predictive_percent/common"
     type: number
-    maximum: 100
```
Comment (wteouchicago) [inline src/gdcdictionary/schemas/other_clinical_attribute.yaml:397]: is there a reason why we are removing the maxes? can percent go over 100?

Comment (wwysoc2) [reply src/gdcdictionary/schemas/other_clinical_attribute.yaml:397]: We found that out yesterday from Megan, apparently these are lung capacity tests and they can go over 100

**Review by wteouchicago — APPROVED** [review]

**Review by wwysoc2 — DISMISSED** [review]
I think hormonal_replacement_therapy_status was left out.  The ticket says "add enum" but I think it's a property we need to add.

Also, did we figure out the pregnancy_count values that were already present?

File: src/gdcdictionary/schemas/other_clinical_attribute.yaml
Comment (stilesj-uchicago) [inline src/gdcdictionary/schemas/other_clinical_attribute.yaml:1]: Am I missing the 'risk_factors','timepoint_category','hormonal_replacement_therapy_status','immunosuppressive_treatment_type', etc enums  that were to be added?

Note: timepoint_category is listed twice in the ticket.

Comment (bullenc) [reply src/gdcdictionary/schemas/other_clinical_attribute.yaml:1]: I see the enums for the properties you listed except for hormonal_replacement_therapy_status. I merged the two separate enum additions in the table in the ticket (`Unknown` and `Sample Procurement`).

**Comment by bullenc** [comment]
> I think hormonal_replacement_therapy_status was left out. The ticket says "add enum" but I think it's a property we need to add.

Ok I will create a new _terms.yaml ticket and circle back to this PR once we include the new property in terms.yaml file

> Also, did we figure out the pregnancy_count values that were already present?

Yes, the only submitted values were for test/QA projects, pretty sure we don't need to migrate those.

File: src/gdcdictionary/schemas/other_clinical_attribute.yaml
Comment (stilesj-uchicago) [reply src/gdcdictionary/schemas/other_clinical_attribute.yaml:1]: I must've gone cross-eyed. I see them now, except for hormonal_replacement_therapy_status

**Review by stilesj-uchicago — APPROVED** [review]

**Pull request merged**